xbmc/windowing/gbm/drm/DRMAtomic.h

```diff
@@ -10,6 +10,10 @@
 
 #include "DRMUtils.h"
 
+#include <deque>
+#include <map>
+#include <memory>
```

Comment:
`uint32_t` and `uint64_t` are added, so `<stdint.h>` should be included here

Reply:
Thanks!